product-service/lib/repositories/products/products-repository.ts

```diff
@@ -1,11 +1,83 @@
-import { products } from './products';
+import { db } from 'lib/db';
 
 export async function getAll(): Promise<Product[]> {
-  return products;
+  const client = db.getClient();
+
+  try {
+    await client.connect();
+
+    const { rows } = await client.query<Product>(`
+      SELECT p.id, p.title, p.description, p.price, p.image_url as "imageUrl", s.count from products p
+        LEFT JOIN stocks s
+        on p.id = s.product_id;
+    `);
+
+    return rows;
+  } finally {
+    await client.end();
+  }
+}
+
+export async function findById(id: string): Promise<Product> {
+  const client = db.getClient();
+
+  try {
+    await client.connect();
+
+    const { rows } = await client.query<Product>(`
+      SELECT p.id, p.title, p.description, p.price, p.image_url as "imageUrl", s.count from products p
+        LEFT JOIN stocks s
+        on p.id = s.product_id
+        WHERE p.id = '${id}';
```

Comment:
Usage of string concatenation/interpolation is not safe in DB queries
https://www.w3schools.com/sql/sql_injection.asp
https://node-postgres.com/features/queries